rpc/src/v1/tests/eth.rs

```diff
@@ -204,6 +204,18 @@ fn eth_get_block() {
 	assert_eq!(tester.handler.handle_request_sync(req_block).unwrap(), res_block);
 }
 
+#[test]
+fn eth_get_block_by_hash() {
```

Comment:
I'm pretty sure this test doesn't actually invoke the light client code (there aren't RPC tests for the light client RPC)

Reply:
Indeed it does not (see [this commit](https://github.com/paritytech/parity/pull/8683/commits/19017661ee8e70821abd20d7ffdc33ba6f19e3e6)) but when I discovered that I had already written the test so I pushed it here anyway. I opened [this](https://github.com/paritytech/parity/issues/8693) to address tests for the light client RPC.